pages/resultset.py

```diff
@@ -13,15 +13,21 @@
 
 class ResultsetPage(Base):
 
+    _job_details_actionbar_locator = (By.ID, 'job-details-actionbar')
     _job_result_status_locator = (By.CSS_SELECTOR, '#result-status-pane > div:nth-child(1) > span')
     _logviewer_button_locator = (By.ID, 'logviewer-btn')
     _resultset_locator = (By.CSS_SELECTOR, 'div.row.result-set')
     _result_status_locator = (By.ID, 'job-details-panel')
+    _unclassified_failure_count_locator = (By.ID, 'unclassified-failure-count')
 
     @property
     def job_result_status(self):
         return self.selenium.find_element(*self._job_result_status_locator).text
 
+    @property
+    def return_unclassified_failure_count(self):
```

Comment:
Nit: Remove `return_` from the property name as all properties should return a value.